dockerfiles/hpcc/templates/dali.yaml

```diff
@@ -23,8 +24,13 @@ spec:
 {{ include "hpcc.addImageAttrs" (dict "root" $ "me" . "imagename" "dali") | indent 8 }}
         volumeMounts:
 {{ include "hpcc.addConfigVolumeMount" . | indent 8 }}
+        - name: dalistore-pv
+          mountPath: "/var/lib/HPCCSystems/dalistore"
```

Comment:
perhaps checkDaliStoreHostMount (and others), and changeHostMountPerms, should be passed the path, so that not hardwired in to _helpers.tpl

Reply:
I want to restructure that whole area